pkg/txnutil/gc/gc_manager.go

```diff
@@ -93,6 +93,8 @@ func (m *gcManager) TryUpdateGCSafePoint(
 		}
 		return nil
 	}
+	m.lastUpdatedTime = time.Now()
```

Comment:
Should L83 `m.lastUpdatedTime = time.Now()` be removed,
Is this a bug in some scenarios?